lib/venue.rb

```diff
@@ -1,2 +1,38 @@
 class Venue
+
+  attr_accessor :name, :capacity, :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def add_patron(patron_name)
+    @patrons << patron_name
+  end
+
+  def yell_at_patrons
+    yelled_patrons = []
+    yelled_patrons << @patrons.map(&:upcase)
+    return yelled_patrons.flatten
+  end
+
+  def over_capacity
+    if @patrons.length > @capacity
+      return true
```

Comment:
You would not need to use an `if/else` here. You are comparing the length of our `@patrons` array to `@capacity` with would return to us a boolean. Hint: re-read the interaction pattern. What is your method name missing?

``` 
pry(main)> venue = Venue.new('Bluebird', 4)
=> #<Venue:0x00007fed7c11dbd8 @capacity=4, @name="Bluebird", @patrons=[]>

pry(main)> venue.add_patron('Mike')
pry(main)> venue.add_patron('Megan')
pry(main)> venue.add_patron('Bob')

pry(main)> venue.over_capacity?
=> false

pry(main)> venue.add_patron('James')
pry(main)> venue.add_patron('Cat')

pry(main)> venue.over_capacity?
=> true
```